superset/utils/log.py

```diff
@@ -194,7 +195,7 @@ def log_with_context(  # pylint: disable=too-many-locals
             slice_id = 0
 
         if log_to_statsd:
-            self.stats_logger.incr(action)
+            stats_logger_manager.instance.incr(action)
```

Comment:
Should the `stats_logger` property on this class be removed as well?

Reply:
nice catch